types/react-router/index.d.ts

```diff
@@ -53,8 +53,7 @@ export interface PromptProps {
   when?: boolean;
 }
 export class Prompt extends React.Component<PromptProps, any> { }
-
-export interface RedirectProps<LocationState = any> {
+export interface RedirectProps<LocationState> {
```

Comment:
We should default to any here. Otherwise this breaks peoples code needlessly. You probably changed this to fix the lint error? Could you try to `tslint:disable-next-line` it instead?

Reply:
Yeah, that was exactly what I was trying to solve - but somehow `npm run test` is not running properly locally, so I just get to see the error when it reaches the CI. 

I'll move it back.

Reply:
@guicamillo

`npm run lint react-router` is probably what you're looking for.